frontend/src/pages/archive/types.ts

```diff
@@ -5,7 +5,8 @@ type CrawlState =
   | "failed"
   | "partial_complete"
   | "timed_out"
-  | "stopping";
+  | "stopping"
+  | "canceled";
```

Comment:
I think this must be an old state, but I had a crawl that returned `canceled` without a `finished` time

Reply:
"Partial Complete" is when a crawl has been manually stopped without finishing the queue yes?  One of my first points of confusion when initially using the site was that I intentionally ended a crawl and wanted the process to be stopped and the data saved but doing this only resulted in a "Partial Complete" status.

Can we rename this to something that better signifies the user's intentions?  "Manually Stopped" or "Manually Ended" perhaps?

Reply:
"Canceled" may also be set from manually ending crawls. It might be that we need to clarify between the "Cancel immediately" and "Stop gracefully" actions. It sounds like what you wanted to do is cancel immediately, which interrupts the crawl and may discard data, vs. stop gracefully which will finish the currently crawled page and retain crawled data (right, @ikreymer?)

If it's important to show to users that a crawl was manually ended, maybe we could add another field `ended_by` with the user ID, so we can show who manually ended the crawl, instead of assuming from the crawl state?

Reply:
Yeah, I think we can come back to this in the separate issue.. 'Partial Complete' is supposed to indicate that not all pages were crawled, but perhaps that can be communicated through page count itself..

Reply:
And yes, I think canceled should now be setting the time when the crawl is canceled.